text/metrics/0238-pipeline-monitoring.md

```diff
@@ -0,0 +1,426 @@
+pipeline having the same `domain` and `signal` attributes, the `name`
+should include additional information to disambiguate the multiple
+instances using the syntax `<type>/<instance>`.  For example, if there
+were two `batch` processors in a collection pipeline (e.g., one for
+error spans and one for non-error spans) they might use the names
+`batch/error` and `batch/noerror`.
+
+#### Pipeline monitoring diagram
+
+The relationship between items received, dropped, and exported is
+shown in the following diagram.
+
+![pipeline monitoring metrics](../images/otel-pipeline-monitoring.png)
+
+### Proposed metrics semantic conventions
+
+The proposed metric names match the following pattern:
+
+| Metric Name               | Meaning                                                            |
+|---------------------------|--------------------------------------------------------------------|
+| `otel.{station}.received` | Inclusive count of items entering the pipeline at a station.       |
+| `otel.{station}.dropped`  | Non-inclusive count of items dropped by a processor in a pipeline. |
+| `otel.{station}.exported` | Inclusive count of items exiting the pipeline at a station.        |
+
+The behavior specified for SDKs and Collectors at each level of detail
+is different, because SDKs do not receive items from a pipeline and
+because they outnumber the other components.
+
+These attributes can be applied to any of the pipeline monitoring
+metrics specified here.
+
+| Attributes     | Meaning                                     | Level of detail (Optional) | Examples                                                   |
+|----------------|---------------------------------------------|----------------------------|------------------------------------------------------------|
+| `otel.signal`  | Name of the telemetry signal                | Basic (Opt-out)            | `traces`, `logs`, `metrics`                                |
+| `otel.name`    | Type, name, or "type/name" of the component | Normal (Opt-out)           | `probabilitysampler`, `batch`, `otlp/grpc`                |
+| `otel.success` | Boolean: item considered success?           | Normal (Opt-out)           | `true`, `false`                                            |
+| `otel.reason`  | Explaination of success/failures.           | Detailed (Opt-in)          | `ok`, `timeout`, `permission_denied`, `resource_exhausted` |
+| `otel.scope`   | Name of instrumentation.                    | Detailed (Opt-in)          | `opentelemetry.io/library`                                 |
+
+For example, when a sampler processor drops an item it may report
+`success=true`, but when a queue processor drops an item it may report
+`success=false`.
+
+#### SDK default configuration
+
+| Metric name         | Enablement level |
+|---------------------|------------------|
+| `otel.sdk.received` | Basic            |
+| `otel.sdk.dropped`  | Normal           |
+| `otel.sdk.exported` | Detailed         |
```

Comment:
I'm struggling to see what metrics I would expect to get out of an SDK with different requirement levels. Would you be able to include an examples with a typical  SDK configuration? I'm imaging an SDK with logs, metrics, and traces enabled, each configured to export data via an OTLP exporter. What metrics and series do I see at Basic, Normal, and Detailed?